lib/team_statistics.rb

```diff
@@ -0,0 +1,310 @@
+class TeamStatistics
+  attr_reader :stat_tracker
+
+  def initialize(stat_tracker)
+    @stat_tracker = stat_tracker
+  end
+# [x] 1. Team Info
+  def team_data_set
+    @stat_tracker[:teams]["team_id"].zip(@stat_tracker[:teams]["franchiseId"], @stat_tracker[:teams]["teamName"], @stat_tracker[:teams]["abbreviation"], @stat_tracker[:teams]["link"])
+  end
+
+  def match_team_id(team_id)
+    team_data_set.map do |info|
+      return info if info[0] == team_id
```

Comment:
I think you can use select here rather than map, take the `return info if` out.

Reply:
return with select: [["12", "26", "Sky Blue FC", "SBL", "/api/v1/teams/12"]] but removing the 'return info if' returns this: {"team_id"=>["1", "23", "Atlanta United", "ATL", "/api/v1/teams/1"], "franchiseId"=>nil, "teamName"=>nil, "abbreviation"=>nil, "link"=>nil}
Left original code. We may be able to tweak this to work.